client/web/antrea-ui/src/routes/summary.tsx

```diff
@@ -23,26 +23,28 @@ import { WaitForAPIResource } from '../components/progress';
 
 type Property = string
 
-const controllerProperties: Property[] = ["Name", "Version", "Pod Name", "Node Name", "Connected Agents", "Healthy", "Last Heartbeat"];
-const agentProperties: Property[] = ["Name", "Version", "Pod Name", "Node Name", "Local Pods", "Node Subnets", "OVS Version", "Healthy", "Last Heartbeat"];
+const controllerProperties: Property[] = ['Name', 'Version', 'Pod Name', 'Node Name', 'Connected Agents', 'Healthy', 'Last Heartbeat'];
+const agentProperties: Property[] = ['Name', 'Version', 'Pod Name', 'Node Name', 'Local Pods', 'Node Subnets', 'OVS Version', 'Healthy', 'Last Heartbeat'];
 
-function refToString(ref: K8sRef): string {
+function refToString(ref: K8sRef | undefined): string {
+    if (!ref) return 'Unknown';
     if (ref.namespace) return ref.namespace + '/' + ref.name;
     return ref.name;
 }
 
 // returns status and last heartbeat time
-function getConditionInfo(conditions: Condition[], name: string): [string, string] {
+function getConditionInfo(conditions: Condition[] | undefined, name: string): [string, string] {
```

Comment:
ditto

Reply:
the 2 mean different things:
```
function getConditionInfo(conditions: Condition[] | undefined, name: string)
```
```
function getConditionInfo(conditions?: Condition[], name: string)
```

The second one is not valid because `?:` indicates an optional parameter, and an optional parameter cannot come before a required parameter (`name`). Here `conditions` is not really an optional parameter, but a required parameter that can be undefined.